charts/meteor-pipelines/templates/byon-import-jupyterhub-image.yaml

```
@@ -0,0 +1,125 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: byon-import-jupyterhub-image
+spec:
+  params:
+    - name: url
+      description: Container image repository url
+      type: string
+    - name: name
+      description: Image name
+      type: string
+    - name: desc
+      description: Custom description
+      type: string
+  tasks:
+    - name: create-imagestream
+      taskRef:
+        name: openshift-client
+        kind: ClusterTask
+      params:
+        - name: SCRIPT
+          value: |
+            echo "Creating image stream:"
+            echo "  Name:       $(context.pipelineRun.name)"
+            echo "  Namespace:  $(context.pipelineRun.namespace)"
+            echo
+            echo "  Image name:        $(params.name)"
+            echo "  Image description: $(params.desc)"
+            echo "  Image URL:         $(params.url)"
+            echo
+            echo "  Phase:             Validating"
+            echo "  Message:           ''"
+            echo "  Visibility:        false"
+            echo
+
+            cat <<EOM | oc apply -f -
```

[UUOC](https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat)? I believe this is better:

```suggestion
            oc apply -f - <<EOM
```

---

That's fair, yup :slightly_smiling_face: